Deployment/templates/continuous-testing.yml

```diff
@@ -91,4 +85,14 @@ steps:
         testRunTitle: $(Environment) Accessibility Test Results
       displayName: 'Publish Accessibility Test Results'
 
+    - task: AzureCLI@2
+      displayName: "Swap MSI BusinessUnit"
+      condition: always()
+      inputs:
+        azureSubscription: "${{ parameters.AzureSubscription }}"
+        scriptType: 'pscore'
+        scriptLocation: 'scriptPath'
+        scriptPath: "$(System.DefaultWorkingDirectory)/Deployment/set_business_unit.ps1"
+        arguments: '-businessUnit $(FileShareService.BusinessUnitMsi) -resourceGroup $(webapp_rg) -webappName $(WEB_APP_NAME)'
```

Comment:
I see this in the Dev library, but not in QA.

Does this variable also need to be added to the QA library, and if so by me?

Reply:
Also, is swapping to the integration test business unit a manual change and re-run in the pipelines?

Reply:
After a good chat with @tejasi15072 and @usha13529, I can now see the logic clearly